scripts/install_xgboost.sh

```diff
@@ -0,0 +1,38 @@
+# Xgboost
```

Comment:
Sheban is not written, is it correct?

Reply:
You're right @eitsupi I should have a shebang here.  Just added one in the last commit.  Thanks!